spec/rubocop/cop/style/string_literals_spec.rb

```diff
@@ -194,6 +194,14 @@
       expect(cop.config_to_allow_offenses).to eq('Enabled' => false)
     end
 
+    it 'registers offense for escaped single quote in single quotes' do
```

Comment:
Would be great if there's a test case for multiple escaped single quotes (and no other escaped characters.) 🙂